pycbc/workflow/grb_utils.py

```diff
@@ -33,7 +33,7 @@
 import os
 import shutil
 import urlparse, urllib
-from glue import segments
+from lal import segments
```

Comment:
Is importing from `lal` a typo here? I just tried it on a PyCBC env from yesterday and it does not work.

Reply:
@titodalcanton Was it an old virtualenv? I.E. before we were installing lalsuite with pip?

Reply:
If so, it might not have had the ligo-segments package. Otherwise, I don't know....

Reply:
I mean, should the import not read `from ligo import segments` like in the rest of the patch?

Reply:
Ah, yes

Reply:
Fixed